Formula/musikcube.rb

```diff
@@ -0,0 +1,42 @@
+class Musikcube < Formula
```

Comment:
> `musikcube contains conflicting version recursive dependencies:`

You'll need to examine the dependency tree and only depend on things using `openssl@3` or `openssl@1.1`, not both.

Reply:
Right, so the problem is `asio` has a hard dependency on `openssl@3`, but **all** other dependencies (e.g. `curl`, `libmicrohttpd`, etc) have a dependency on `openssl@1.1`.

This is why I had originally pulled `asio` in as a git submodule, instead of a `brew` dependency -- but I was asked earlier in this thread to change that. Is there some sort of way for me to request a dependency use a different version of a transitive dependency? Or perhaps create a new version of `asio` that depends on `openssl@1.1`?